<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFilmWidget</class>
 <widget class="QWidget" name="AddFilmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add film</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Avatar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yearEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>2009</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Rate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rateEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>7.8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Votes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="votesEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>1000000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,5">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="genreListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>1212412</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="topMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="durationEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>160</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Film</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Series</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Certificate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="certificateSelect">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Certificate</string>
         </property>
         <item>
          <property name="text">
           <string>TV-Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TV-Y7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TV-PG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PG-13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TV-14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TV-MA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NC-17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Nudity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="nuditySelect">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mild</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moderate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Severe</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Violence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="violenceSelect">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mild</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moderate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Severe</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1">
     <property name="topMargin">
      <number>16</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDial" name="dial">
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
